cluster-autoscaler/main.go

```diff
@@ -211,6 +211,7 @@ var (
 	skipNodesWithLocalStorage          = flag.Bool("skip-nodes-with-local-storage", true, "If true cluster autoscaler will never delete nodes with pods with local storage, e.g. EmptyDir or HostPath")
 	minReplicaCount                    = flag.Int("min-replica-count", 0, "Minimum number or replicas that a replica set or replication controller should have to allow their pods deletion in scale down")
 	nodeDeleteDelayAfterTaint          = flag.Duration("node-delete-delay-after-taint", 5*time.Second, "How long to wait before deleting a node after tainting it")
+	scaleDownNodeCategorizationTimeout = flag.Duration("scale-down-node0categorization-timeout", 5*time.Minute, "How long should we categorize nodes as needed/unneeded in scale down logic.")
```

Comment:
nit: wouldn't "scale down simulation" be slightly more understandable than "scale down node categorization" at the flag level?

Also, there's a typo in the flag name (s/0/-/).

Reply:
renamed